docs/reference.md

```diff
@@ -48,7 +48,10 @@ that Slack should not retry that request, you should set `responseOptions.failWi
 truthy value. If you want Slack to send the request to a different URL, you should set the
 value of `responseOptions.redirectLocation` to the URL you want. Lastly, if you'd like to
 set the body of the response, you should set the `responseOptions.content` value to a String of
-body content.
+body content. EventEmitters allow you to add many handlers, but its recommended that when using the
```

Comment:
Since this is a diff for grammar and spelling --> it's recommended that when using the waitForResponse option you only (add the apostrophe on its and delete either one of the "that"s